classes/metric/active_users_metric.php

```diff
@@ -55,4 +55,13 @@ public function get_metric_item(): metric_item {
         $users = $DB->count_records_select('user', 'lastlogin > ?', [$now - $this->get_time_window()]);
         return new metric_item($this->get_name(), $now, $users, $this);
     }
+
+    /**
+     * Metric's ability to be backfilled.
+     *
+     * @return bool
+     */
+    public function is_backfillable(): bool {
```

Comment:
If this is going to appear in every metric, then it should also be put into the base class, even if it is abstract.

Reply:
Fixed